tests/test_parallel_extractors.py

```
@@ -149,51 +148,64 @@ def test_merge_keeps_distinct_non_recurring_commitments_under_same_origin_ref():
-    assert elapsed < delay * len(EXTRACTION_MODULES), (
-        f"wall-clock {elapsed:.2f}s 未短于串行总和 {delay*len(EXTRACTION_MODULES):.2f}s")
 
 
 def test_serial_extract_stays_serial(read_game, monkeypatch):
-    """parallel=False（形态1/api 默认）峰值并发==1，串行不受影响。"""
+    """parallel=False（形态1/api 默认）在调用线程串行跑模块，峰值并发==1。
+
+    忠实证明：run 窗口内计数 + 调用线程同一性。旧夹具把 ++/max/-- 放同一锁块且
+    run 在计数外，生产忽略 parallel 走 ThreadPool 后串行案仍绿。
+    """
     db, state, content = read_game
```

Ruff (0.16.3)

[warning] Unpacked variable `content` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)